src/data/data-table/wds/wds-status.test.ts

```diff
@@ -135,6 +135,7 @@ describe('useWdsStatus', () => {
       },
       accessScope: 'WORKSPACE_SHARED',
       labels: {},
+      region: '',
```

Comment:
Can apps have an empty string as a region?

Reply:
I set it to the default